extensions/expression-language/ProcessContextExpr.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <core/ProcessContext.h>
+
+namespace org {
+namespace apache {
+namespace nifi {
+namespace minifi {
+namespace core {
+
+bool ProcessContext::getProperty(const std::string &name, std::string &value,
+                                 const std::shared_ptr<FlowFile> &flow_file) {
+  if (expressions_.find(name) == expressions_.end()) {
```

Comment:
where is the expressions_ map populated?

Reply:
I did not see the code path for EL for property handling. it will make sense to change README to illustrate the example use case for EL

Reply:
Good catch. I think it may not be populated. Will fix.

Reply:
...actually you'll notice a few lines down:

```c++
expressions_.emplace(name, expression::compile(expression_str));
```

So it's def. populating the map.

Reply:
OK.